### PR TITLE
add CWE and CWE_url

### PR DESCRIPTION
**Pull request by SPoint42**

### CODE REVIEW

File: SecurityCodeScan/Config/Messages.yml [hunk 1]
```diff
@@ -19,140 +19,202 @@ SCS0001:
...
 SCS0018:
   title: "Potential Path Traversal vulnerability was found where '{0}' in '{1}' may be tainted by user-controlled data from '{2}' in method '{3}'."
   description: The untrusted value passed to the redirect should be validated.
+  cwe: CWE-22
+  cwe_url: https://cwe.mitre.org/data/definitions/22.html
 
 SCS0019:
   title: OutputCache annotation is disabling authorization checks.
   description: Having the annotation [OutputCache] will disable the annotation [Authorize] for the requests following the first one.
+  cwe: N/A
```
Comment (JarLob) [inline SecurityCodeScan/Config/Messages.yml]:
```suggestion
  cwe: CWE-524
```

File: SecurityCodeScan/Config/Messages.yml [hunk 2]
```diff
@@ -19,140 +19,202 @@ SCS0001:
...
+  cwe_url: N/A
```
Comment (JarLob) [inline SecurityCodeScan/Config/Messages.yml]:
```suggestion
  cwe_url: https://cwe.mitre.org/data/definitions/524.html
```

**Comment by JarLob** [comment]
Sorry for delay. Cleaned up the mess in website links. You need it only for the standalone tool, correct? I'll add a command line parameter, something like --cwe to print it as you suggested.
I'm little hesitant to add it to a standard output as it may introduce a visual bloat like `filenamehere.cs(64,25,64,48): warning SCS0028: CWE061: Bla bla bla`

**Comment by SPoint42** [comment]
> Sorry for delay. Cleaned up the mess in website links. You need it only for the standalone tool, correct? I'll add a command line parameter, something like --cwe to print it as you suggested.
> I'm little hesitant to add it to a standard output as it may introduce a visual bloat like `filenamehere.cs(64,25,64,48): warning SCS0028: CWE061: Bla bla bla`

I think it could be useful for a lot of guys for the CWE defintion. But, Yes, for the CI/CD integration it really need and a --cwe parameter is OK

**Comment by JarLob** [comment]
Closes #184

**Pull request merged**